packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1527,6 +1531,7 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
       return Tooltip(
         message: widget.tooltip ?? MaterialLocalizations.of(context).showMenuTooltip,
         child: InkWell(
+          borderRadius: widget.borderRadius,
```

Comment:
You are going to need a test for this. What about a `testWidgets()` that:
1) Pumps a single `PopupMenuButton`
2) Finds the `InkWell` that is included in the widget tree (as part of the PopupMenuButton)
3) Compares the ink well's border radius in an `expect()`
3a) You might want to test with `null` and a specific border radius value, so that would be a test for each